clang/lib/Basic/Targets/WebAssembly.cpp

```diff
@@ -307,6 +310,14 @@ bool WebAssemblyTargetInfo::handleTargetFeatures(
       HasReferenceTypes = false;
       continue;
     }
+    if (Feature == "+gc") {
+      HasGC = true;
+      continue;
+    }
+    if (Feature == "-gc") {
+      HasGC = false;
+      continue;
+    }
```

Comment:
When we add new feature like this don't we normally add them to `WebAssemblyTargetInfo::initFeatureMap` in `addBleedingEdgeFeatures`.. I would also expect the test for bleeding edge CPU to be updated, right?

Reply:
the test for llvm/test/CodeGen/WebAssembly/target-features-cpus.ll below does seem to include gc in bleeding-edge. But you're right, that we've been also adding things with `addBleedingEdgeFeatures` so it's weird that that test changes. We should look into what's going on there.